modules/mauth-authenticator-akka-http/src/test/scala/com/mdsol/mauth/akka/http/MAuthDirectivesSpec.scala

```diff
@@ -184,4 +196,229 @@ class MAuthDirectivesSpec extends WordSpec with Matchers with ScalatestRouteTest
       }
     }
   }
+
+  "authenticate for V2" should {
+    lazy val route: Route = authenticate(executor, authenticatorV2, timeout, requestValidationTimeout).apply(complete(HttpResponse()))
+    val publicKey = MAuthKeysHelper.getPublicKeyFromString(FixturesLoader.getPublicKey)
+
+    "pass successfully authenticated request with parameters" in {
```

Comment:
"...with both V1 and V2 headers ,with V2 headers taking precedence"

To test precedence works, please use invalid values for v1 headers.

Reply:
updated as suggested https://github.com/mdsol/mauth-jvm-clients/pull/92/commits/735bd0daeb6afd8ec05d1562413a7721f033797f